lms/djangoapps/ccx/utils.py

```diff
@@ -128,6 +128,19 @@ def get_ccx_for_coach(course, coach):
     return None
 
 
+def get_ccx_for_coach_by_ccx_id(course, coach, ccx_id):
```

Comment:
so I understand why you are filtering by `coach`, but what's the reason to filter for `course.id`? To verify that the course is the master course of this ccx?